README.md

```diff
@@ -9,26 +9,338 @@ JavaScript lodash Templates
+  var comment = document.getElementById("commentText").value;
+  //insert comment into "comments" div in this format:
+  //<div class="comment"><p>comment</p><p>Posted By: <span class="commenter">commenter</span></p></div>
+
+  //create template string
+  var commentTemplate = '<div class="comment"><p><%= comment %></p><p>Posted By: <span class="commenter"><%= commenter %></span></p></div>';
+  //create template function
+  var templateFn = _.template(commentTemplate);
+
+  var commentsDiv = document.getElementById("comments");
+
+  //execute template function with JSON object for the interpolated values
+  var templateHTML = templateFn({ 'comment': comment, 'commenter': commenter });
+
+  //append rather than replace!
+  commentsDiv.innerHTML += templateHTML;
+}
+```
+
+Reload your `index.html` and see it in action!
+
+Okay, this is all fine and good, and we aren't concatenating strings
+anymore, but we're still building out a huge string for the template,
+and this is a pretty simple template. What happens when we have a lot of
+markup to work with?
+
+### HTML Templates in JavaScript
+
+Putting our templates in a string isn't great for a couple of reasons.
+We don't get the human-readable, indented structure of regular HTML. We
+don't get the syntax highlighting on the tags that our editor provides.
+Adding or changing HTML inside a string is difficult and prone to error.
+What we need is a way to use a regular chunk of HTML as our template.
+
+Fortunately, lodash gives us a great way to do this. We can use a
+special `<script>` tag with the content-type of `text/x-lodash-template`
+and embed HTML directly inside. Let's redo our comment template.
+
+This time, we're going to put this in the bottom of our `index.html`
+like it was any other JavaScript block.
+
+```html
+<!-- index.html, place before end body tag -->
+<body...>
+  <article>...</article>
+  <script id="comment-template" type="text/x-lodash-template">
+    <div class="comment">
+      <div class="comment-body"><%= comment %></div>
+      <div class="commenter">
+        <p>
+          <span class="posted-by">Posted By: </span>
+          <%= commenter %>
+        </p>
+      </div>
+    </div>
+  </script>
+</body>
+```
+
+Inside that `<script id="comment-template">` tag we just have *mostly*
+regular HTML! The only difference is, since we're inside a `script` tag,
+we also can use the interpolation delimiters from lodash to mark where
+our dynamic data is.
+
+Okay, we've made a significant change here, so surely there's a lot of
+work require to wire this up, right?
+
+```js
+// index.js
+function postComment() {
+  var commenter = document.getElementById("commenterName").value;
+  var comment = document.getElementById("commentText").value;
+
+  //insert comment into "comments" div in this format:
+  //<div class="comment"><p>comment</p><p>Posted By: <span class="commenter">commenter</span></p></div>
+
+  //create template string
```

Comment:
yeah I went back and forth and ultimately thought it was more illustrative to comment the old `commentTemplate` line, but I could cut it too.